packages/testcontainers/__tests__/containers/NativeChainContainer/Coinbase.test.ts

```diff
@@ -0,0 +1,118 @@
+import { Network } from 'testcontainers'
+import { NativeChainContainer, StartedNativeChainContainer } from '../../../src'
+import waitForExpect from 'wait-for-expect'
+
+describe('coinbase maturity faster by time travel', () => {
+  let container: StartedNativeChainContainer
+
+  beforeAll(async () => {
+    const startedNetwork = await new Network().start()
+    container = await new NativeChainContainer()
+      .withNetworkMode((startedNetwork).getName())
```

Comment:
```suggestion
    const network = await new Network().start()
    container = await new NativeChainContainer()
      .withNetwork(network)
```

You can do this now with testcontainers v9.